vars/runIntegTests.groovy

```diff
@@ -0,0 +1,24 @@
+void call(Map args = [:]) {
+    git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
+
+    def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    String testManifest = args.testManifest ?: "manifests/${TEST_MANIFEST}"
+
+    String tarball = args.architecture == "x64" ? env.ARTIFACT_URL_linux_x64 : env.ARTIFACT_URL_linux_arm64
+    String buildManifest = args.architecture == "x64" ? env.BUILD_MANIFEST_URL_linux_x64 : env.BUILD_MANIFEST_URL_linux_arm64
+    echo "buildManifest: ${buildManifest}"
+    echo "testManifest: ${testManifest}"
+
+    if (tarball == null) {
+        echo "Skipping integ tests for ${args.architecture} architecture because ${args.architecture} artifact was not built."
+    } else {
+        IntegTests: {
+            // TODO: will need to override integ-test in Jenkins with the new workflow in jenkins/opensearch/integ-test.jenkinsfile
+            build job: 'Playground/ohltyler-integ-test',
+            parameters: [
+                string(name: 'TEST_MANIFEST', value: "${testManifest}"),
+                string(name: 'BUILD_MANIFEST_URL', value: "${buildManifest}"),
+            ]
```

Comment:
We should add the parameter `wait: false` so that the build workflow isn't blocked
We should also add the parameter `propagate: false` so if the tests pass/fail it doesn't impact the distribution-build job.

Reply:
Similar to my question [here](https://github.com/opensearch-project/opensearch-build/pull/1429#discussion_r782623755), how do we want to go about test failures/successes impacting the jenkins jobs?

I see 2 main paths forward:
1. Fail the step if the tests failed, which would cause the entire job to fail
2. Jenkins job should always succeed (assuming no syntax / other workflow-related bugs), and test success/failure is handled via notifications and/or other jenkins jobs that _will_ fail if the tests fail

Reply:
1. I put a more detail reply on that first question, for now only job failure == workflow failure is more useful because we are using as we are building so we are still focused on stability of the workflow

2. As you've written it I think so.  Note; I think we can revisit this when we have an automated action like 'deploy to production' that has the required pre-reqs run.   I think we would want to make sure things like 8+ hour long build jobs are still correcting rolling up state across different workflows.

Reply:
Right, I think that makes a lot of sense. We can follow this fail-fast strategy for now.